.travis.yml

```diff
@@ -39,9 +39,12 @@ jobs:
 
 deploy:
   - provider: script
-    script: bash ci_tools/publish.sh "$fbn_pypi_username" "$fbn_pypi_password"
+    script: >-
+            bash ci_tools/bumpversion.sh &&
+            bash ci_tools/publish.sh "$fbn_pypi_username" "$fbn_pypi_password"
     on: master
 
+
 notifications:
   slack:
     on_success: never
```

Comment:
It might be worth posting a slack message when we publish a new version

Reply:
Sure.
Into which chat room? I see the YAML references an encoded room.

Reply:
#build_public is probably ok for now

Reply:
Sure, updated.